pom.xml

```diff
@@ -56,6 +56,11 @@
         <embeddedMongoDbLogDir>${project.build.directory}/embeddedMongoDb/logs</embeddedMongoDbLogDir>
         <!-- Not using default port to reduce chances of shutting down an external MongoDB instance -->
         <embeddedMongoDbPort>27018</embeddedMongoDbPort>
+
+        <!-- Since we require users to import a specific Hibernate Search module explicitly,
+         all integration tests need to know which version that shall be.
+         So the property needs to be defined in the global parent pom -->
+        <hibernate-search.module.slot>5.1</hibernate-search.module.slot>
```

Comment:
it was 5.0, and since the JIRA is about moving to 5.1 I changed it.
Also the versions were hardcoded in the tests, now they are properly injected from the build.

The aspect on module distribution doesn't change.